service/api.go

```diff
@@ -384,6 +375,28 @@ func (as *APIServer) FetchExpansionsForTask(w http.ResponseWriter, r *http.Reque
 	gimlet.WriteJSON(w, res)
 }
 
+func accessAdminVars(t *task.Task, v *model.Version) (bool, error) {
```

Comment:
I was thinking instead that GetAdminOnlyVars() could handle this. It looks like it would just need to take in the task, since task also has the requester field. (I think this should check IsSystemActivator instead of just repotracker). This would make it harder for future developers to add AdminOnlyVars accidentally.

Reply:
got rid of the stepback logic because it should automatically haev access if its system activated 
isadmin function cant be imported so i replaced it